### PR TITLE
Fromcvmfs

### PR DESCRIPTION
**Pull request by clementhelsens**
edm4hep updated to not use local fccanalyses

### CODE REVIEW

**Comment by clementhelsens** [comment]
@EmanuelPerez @patriziaazzi 
new PR that super seeds #60. It does not required to locally download and compile FCCAnalyses.

@vvolkl @gganis we need https://github.com/HEP-FCC/FCCAnalyses/releases/tag/v0.3.3 as soon as possible in the stack, as a performance issue has been fixed since last version

**Comment by vvolkl** [comment]
@clementhelsens that's there already (as of `2021-06-04`) maybe we should start sending out emails for the releases?

File: fast-sim-and-analysis/FccFastSimAnalysis.md [hunk 1]
```diff
@@ -16,78 +16,277 @@ This tutorial will teach you how to:
 
 ## Part II: Analyse with FCCAnalyses
 
-For this part we start by cloning the FCCAnalyses GitHub repository
+We stay in the same directory where we have produced our samples with ```DelphesPythia8_EDM4HEP```.
+
+For this part we start by making sure we can use FCCAnalyses by checking we can load the library correctly:
 
 ```bash
```
Comment (vvolkl) [inline fast-sim-and-analysis/FccFastSimAnalysis.md:23]:
```suggestion
```python
```

File: fast-sim-and-analysis/FccFastSimAnalysis.md [hunk 2]
```diff
@@ -16,78 +16,277 @@ This tutorial will teach you how to:
...
 
-This tutorial runs with fcc-edm so we need to checkout the corresponding branch
+if no error message, it means we can load it.
+
+Alternatively, you can check with spack, replacing ```<version>``` by the key4hep version you have set up.
 
 
 ```bash
-git checkout fccedm
+spack find -p -d key4hep-stack@<version> | grep fccanalyses
```
Comment (vvolkl) [inline fast-sim-and-analysis/FccFastSimAnalysis.md:36]: or 
`echo $FCCANALYSES`

**Comment by clementhelsens** [comment]
@vvolkl , this is great! and yes setting up emails when new release is deployed seems excellent! I'm not sure hte main fccsw list is good for that, neither the key4hep one, and we might create a dedicated list, in key4hep most likely, like key4hep-sw-dev? or key4hep-sw-rel?

File: fast-sim-and-analysis/FccFastSimAnalysis.md (hunk 2 above)
Comment (clementhelsens) [reply fast-sim-and-analysis/FccFastSimAnalysis.md:36]: I did not checked it you changed the path of this env var, as I didn't wanted it to point on  a non existing dir as it was the case last week.

**Pull request merged**